.github/workflows/checks.yml

```diff
@@ -3,6 +3,7 @@ on:
     types:
       - opened
       - edited
+      - reopened
```

Comment:
Why do you precise the `types` of pull request? 🤔 Just being curious!

Reply:
Whenever the PR is updated (e.g adjust the title) then the checks rerun again. I modify because I was in trouble with this PR title is not being updated at `Check title job`